tools/build_utils.py

```diff
@@ -101,18 +103,30 @@ def cmake_build(build_dir, src_location, cmake_flags, verbose):
 
     cmake_cmd = ["cmake"]
     cmake_cmd.extend(cmake_flags)
-    cmake_cmd.extend([src_location])
+    if (platform.system() == 'Windows'):
+        cmake_cmd.extend([src_location.replace("\\", "\\\\")])
+    else:
+        cmake_cmd.extend([src_location])
 
     command_executor(cmake_cmd, verbose=True)
 
     import psutil
     num_cores = str(psutil.cpu_count(logical=True))
-    cmd = ["make", "-j" + num_cores]
-    if verbose:
-        cmd.extend(['VERBOSE=1'])
-    command_executor(cmd, verbose=True)
-    cmd = ["make", "install"]
-    command_executor(cmd, verbose=True)
+    if (platform.system() == 'Windows'):
+        cmd = [
```

Comment:
Maybe add a TODO for debug builds later?

Reply:
done